cmd/shortener/main.go

```diff
@@ -1,3 +1,64 @@
 package main
 
-func main() {}
+import (
+	"fmt"
+	"github.com/go-chi/chi/v5"
+	"github.com/google/uuid"
+	"io"
+	"log"
+	"net/http"
+)
+
+var (
+	shortURL = map[string]string{
```

Comment:
Please use additional abstraction such as `storage`

Reply:
ok

Reply:
I mean, you don't need to use a global variable. You could move it to the `storage` package.

Reply:
moved shortURL in localStorage